components/engine/src/errors.rs

```diff
@@ -16,8 +15,8 @@ quick_error! {
         NotInRange( key: Vec<u8>, regoin_id: u64, start: Vec<u8>, end: Vec<u8>) {
             description("Key is out of range")
             display(
-                "Key {:?} is out of [region {}] [{:?}, {:?})",
-                escape(&key), regoin_id, escape(&start), escape(&end)
+                "Key {} is out of [region {}] [{}, {})",
+                hex::encode_upper(&key), regoin_id, hex::encode_upper(&start), hex::encode_upper(&end)
```

Comment:
Should the key be surrounded by `"`?

Reply:
Maybe not necessary. We are not wrapping them by `"` as well in the protobuf output.